src/main/kotlin/com/terraformation/backend/seedbank/api/AccessionsController.kt

```diff
@@ -85,11 +85,15 @@ data class SeedQuantityPayload(
                 "the \"units\" field is \"Seeds\". This is always calculated on the server side " +
                 "and is ignored on input.",
         readOnly = true)
-    val grams: BigDecimal? = null
+    val grams: BigDecimal? = null,
+    @Schema(description = "Notes associated with this quantity when being changed by end user.")
+    val notes: String? = null,
```

Comment:
Added notes to the remaining quantity payload, even though it's only plumbed into updating the history. Notes are not stored as part of the accession payload (no client requirement so far).
Would it be better to extract this property out of the remaining quantity payload? That would require more explicit plumbing of notes elsewhere.

Reply:
Yeah, this feels like the wrong place to me since there are several quantities on an accession and notes only apply to the remaining quantity, not, e.g., the subset quantity.

Maybe `UpdateAccessionRequestPayloadV2` should have a field for remaining quantity notes.